src/Inventory.java

```diff
@@ -74,7 +72,7 @@ public void update() throws FileNotFoundException {
         writer.println("product_id,quantity,wholesale_cost,sale_price,supplier_id");
         Iterator<Product> itr = iterator();
         while (itr.hasNext()) {
-            writer.println(itr.next());
+            writer.print(itr.next());

```

Comment:
I listed the print function instead of println as an approach to possibly fix the issue. It did not.